modules/build/src/test/scala/scala/build/tests/BuildTests.scala

```diff
@@ -511,23 +533,23 @@ class BuildTests extends munit.FunSuite {
       )
     }
   }
-
-  test("Pass files with only commented directives as is to scalac") {
-    val testInputs = TestInputs(
-      os.rel / "Simple.scala" ->
-        """// using lib com.lihaoyi::pprint:0.6.6
-          |object Simple {
-          |  def main(args: Array[String]): Unit =
-          |    pprint.log("Hello " + "from tests")
-          |}
-          |""".stripMargin
-    )
-    testInputs.withBuild(defaultOptions, buildThreads, bloopConfig) { (_, _, maybeBuild) =>
-      val sources = maybeBuild.toOption.get.successfulOpt.get.sources
-      expect(sources.inMemory.isEmpty)
-      expect(sources.paths.lengthCompare(1) == 0)
-    }
-  }
+//
```

Comment:
Commented out test?

Reply:
It's a test that asserts that sourcefile with directives in comments is not modified. Currently, there's no information about that from my library. We can add it or just always cut the directives part from a file

Reply:
I think it should not block the code now but I want to avoid source generation as much as possible so I think the library should expose this information as a follow up - since in future we want to warn if `@using` and `// using` style is mixed

Reply:
It'd be nice to have a way to preserve those tests.